pep-0001.txt

```diff
@@ -547,7 +547,7 @@ optional and are described below.  All other headers are required.
     Status: <Draft | Active | Accepted | Provisional | Deferred | Rejected |
              Withdrawn | Final | Superseded>
     Type: <Standards Track | Informational | Process>
-  * Content-Type: <text/x-rst | text/plain>
+  * Content-Type: text/x-rst
```

Comment:
This is saying that `text/plain` is no longer a valid value, which I don't think is correct.  It's redundant but should continue to be accepted.  Or are you saying that `text/plain` actually *will no longer* be accepted?  (That's different than what the description implies.)

Reply:
> `text/plain` actually will no longer be accepted

The status quo is that all PEPs are `text/x-rst`, and should PEP 676 be approved `text/plain` would not work (of course technically it could and support could be added, but no-one has asked for this).

I would support dropping support for `text/plain` entirely.

A

Reply:
It has not been a valid value for actual PEPs committed to this upstream PEPs repo for perhaps a year now, since the linters have specifically checked that `Content-Type` is `text/x-rst` and fail if it is not. It is still mentioned as a historical note in PEP 1, but is not documented in the documentation aimed at PEP authors, since it has no obvious utility for authors of any new or existing PEP, just for historical purposes.

Given, if PEP 676 is accepted, the ancient `pep2html.py` script will be retired anyway and our linters already enforce it, I don't think its worth the effort to go through and actually strip it out (and the small but non-trivial risk that would entail).